ui/desktop/electron-app/src/helpers/spawn-promise.js

```diff
@@ -42,25 +33,37 @@ module.exports = {
       let outputStream = '';
       let errorStream = '';
 
-      childProcess.stdout.on('data', (data) => {
+      const processData = (data) => {
         outputStream += data.toString();
         const jsonData = jsonify(outputStream);
-        if (jsonData) resolve(jsonData);
-      });
+        if (jsonData) {
+          childProcess.removeListener('data', processData);
```

Comment:
Probably want to remove both event listeners (including error)?  And remove both in the error handler below too.

Reply:
Good point. Will update.

Reply:
Updated to remove all listeners before resolving/rejecting promise.